coop_shift/model/shift_shift.py

```diff
@@ -394,15 +374,12 @@ def _compute_end_date_fields(self):
                 "You can not create Shift if your timezone is not defined."))
         for shift in self:
             if shift.date_end:
-                shift.end_date_for_mail = datetime.strftime(
-                    fields.Datetime.from_string(shift.date_end),
-                    "%d/%m/%Y %H:%M")
+                shift.end_date_for_mail = \
+                    datetime.strftime(shift.date_end, "%d/%m/%Y %H:%M")
                 shift.end_time_for_mail = datetime.strftime(
```

Comment:
Same for this field.
It shouldn't be necessary if we use `format_datetime` on the template.

Reply:
@ivantodorovich : this field not used in email template at lest not in coop_shift module

Reply:
Can you search for usage on all the repository? to see if it's used by other modules?
If it's not used, just remove it